### PR TITLE
tox: test mimic deployment

### PR DESCRIPTION
**Pull request by leseb**
Let's try to deploy mimic.

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-filestore_osds_non_container

File: tox.ini [hunk 1]
```diff
@@ -158,6 +158,11 @@ setenv=
   luminous: CEPH_DOCKER_IMAGE_TAG_BIS = latest-bis-luminous
   luminous: UPDATE_CEPH_STABLE_RELEASE = luminous
   luminous: UPDATE_CEPH_DOCKER_IMAGE_TAG = latest
+  mimic: CEPH_STABLE_RELEASE = mimic
+  mimic: CEPH_DOCKER_IMAGE_TAG = latest-mimic
+  mimic: CEPH_DOCKER_IMAGE_TAG_BIS = latest-bis-mimic
```
Comment (leseb) [inline tox.ini]: s/latest-bis-mimic/latest-bis/

**Comment by guits** [comment]
Closing in favor of #2670

**Pull request closed without merging**